heareval/tasks/config/speech_commands.py

```diff
@@ -0,0 +1,36 @@
+"""
+Configuration for the google speech commands task
+"""
+
+# TODO: move some of these to a global config and import that here
+# See: https://github.com/neuralaudio/hear2021-eval-kit/issues/10
+
+TASKNAME = "speech_commands-v0.0.2"
+
+# Number of CPU workers for Luigi jobs
```

Comment:
This should be a generic luigi config I think

Reply:
Added a global dataset_configy.py file